### PR TITLE
How to contribute to Koma docs

### PR DESCRIPTION
**Pull request by Stockless**
Added in 'how-to' the document '5-contribute-to-koma.md' with the instructions in case an external developer wants to contribute to the project or report an issue.
This change includes its respective assets and the redirecton of 'Contribution Guidelines" in README.md

Continuation of #382.

### CODE REVIEW

File: docs/src/how-to/5-contribute-to-koma.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+This command will clone KomaMRI.jl's repository (`dev` version) to your `~/.julia/dev/KomaMRI/` directory.
+
+### 2. Create your Fork of KomaMRI
+
+You need to create a fork of the KomaMRI repository in GitHub. This allows you to have your own copy of the repository to work on without modifying the original repository.
+
+Go to the official [KomaMRI repository](https://github.com/JuliaHealth/KomaMRI.jl) and follow the steps below:
+
+![](../assets/create-fork-step1.png)
+![](../assets/create-fork-step2.png)
```
Comment (cncastillo) [inline docs/src/how-to/5-contribute-to-koma.md:32]: Information from https://github.com/JuliaHealth/KomaMRI.jl/pull/382#discussion_r1576804595 is not included, like the access denied error. Also, please take a look at the images I uploaded.

File: docs/src/how-to/5-contribute-to-koma.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Before you report an isue, we assume that you have read the available [Documentation](https://cncastillo.github.io/KomaMRI.jl) and recomend you to to search for existing [Issues](https://github.com/cncastillo/KomaMRI.jl/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue.
+
+If you then still feel the need to report an issue, we recommend the following:
+
+- Open an [Issue](https://github.com/cncastillo/KomaMRI.jl/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions, depending on what seems relevant.
+
+We will then take care of the issue as soon as possible.
```
Comment (cncastillo) [inline docs/src/how-to/5-contribute-to-koma.md:142]: No need to add this.

**Review by cncastillo — CHANGES_REQUESTED** [review]
Multiple changes to the images are required. Also, please be consistent with the capitalization of the titles so it matches the rest of the docs (no title case). Also, you didn't mention the previous PR #382. 

As this is related to the documentation you need to label the PR with "documentation".

File: docs/src/how-to/5-contribute-to-koma.md (hunk 1 above)
Comment (Stockless) [reply docs/src/how-to/5-contribute-to-koma.md]: changed in commit [f13e4a2](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/f13e4a293e53687536a83eb6d9f6ab00d8dffc02)

File: docs/src/how-to/5-contribute-to-koma.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    - "GitHub Pull Requests" for collaboration with GitHub.
+
+## Installing KomaMRI as a developer
+### 1. Clone KomaMRI repository
+
+To install the dev version of Koma, we will use the Julia REPL:
+```julia-repl
+pkg> dev KomaMRI
+``` 
+This command will clone KomaMRI.jl's repository (`dev` version) to your `~/.julia/dev/KomaMRI/` directory.
```
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/how-to/5-contribute-to-koma.md]: changed in commit [655335d](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/655335d219719c4b372575ce38a7d0fa24d0e71d)

File: docs/src/how-to/5-contribute-to-koma.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+Before you begin, it's necessary to install and configure a few essential tools on your machine to ensure a smooth development experience:
+
+- **Juliaup:** This is the manager for installing different versions of Julia.
+- **Julia:** This is the programming language. It is advisable to install it with Juliaup.
+- **Git:** This is a version control system handy for coding.
+- **GitHub:** This is a cloud-based Git repository handy for managing the KomaMRI project. You need to create an account.
+- **VScode:** This is a code editor with support for development operations.
+- **VScode Plugins:** They enable VSCode to have more handy features. We recommend you install the following: 
+    - "Julia" for Julia development
+    - "GitHub Pull Requests" for collaboration with GitHub.
```
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/how-to/5-contribute-to-koma.md]: changed in commit [9a05d6a](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/9a05d6a76e2523e78b1f4ea9df8893d7a5a32c46)

File: docs/src/assets/add-remote.png
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/assets/add-remote.png:1]: Cropped the image for better visualization in commit [a75357c](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/a75357c9ff82c7a7a7fb0185250425da39d295f3)

File: docs/src/assets/create-branch-step1.png
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/assets/create-branch-step1.png:1]: Cropped the image for better visualization in commit [a75357c](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/a75357c9ff82c7a7a7fb0185250425da39d295f3)

File: docs/src/assets/create-fork-step1.png
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/assets/create-fork-step1.png:1]: Cropped the image for better visualization in commit [a75357c](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/a75357c9ff82c7a7a7fb0185250425da39d295f3)

File: docs/src/assets/create-fork-step2.png
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/assets/create-fork-step2.png:1]: Cropped the image for better visualization in commit [a75357c](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/a75357c9ff82c7a7a7fb0185250425da39d295f3)

File: docs/src/assets/create-pull-request.png
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/assets/create-pull-request.png:1]: Cropped the image for better visualization in commit [a75357c](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/a75357c9ff82c7a7a7fb0185250425da39d295f3)

File: docs/src/assets/create-remote-step1.png
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/assets/create-remote-step1.png:1]: Cropped the image for better visualization in commit [a75357c](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/a75357c9ff82c7a7a7fb0185250425da39d295f3)

File: docs/src/how-to/5-contribute-to-koma.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cncastillo) [reply docs/src/how-to/5-contribute-to-koma.md]: I believe that the paths for Windows need the backslashes.

File: docs/src/how-to/5-contribute-to-koma.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (cncastillo) [reply docs/src/how-to/5-contribute-to-koma.md]: Much better, but the first paragraph is a little verbose.

File: docs/src/how-to/5-contribute-to-koma.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/how-to/5-contribute-to-koma.md]: In the commit [6dc61ca](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/6dc61ca3f7f3a947b46cb7df06660643d1c3ee2a) I fixed the backslashes and added the mention of this path in "4. Open your forked repository in VSCode"

File: docs/src/how-to/5-contribute-to-koma.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Stockless) [reply docs/src/how-to/5-contribute-to-koma.md]: changed it in commit [463b2cb](https://github.com/JuliaHealth/KomaMRI.jl/pull/509/commits/463b2cb57bd24b61e8994e1e62ff8f92ec1af1f8)

**Pull request merged**